x-pack/plugins/security/server/saved_objects/saved_objects_security_extension.ts

```diff
@@ -38,12 +71,115 @@ export class SavedObjectsSecurityExtension implements ISavedObjectsSecurityExten
   private readonly auditLogger: AuditLogger;
   private readonly errors: SavedObjectsClient['errors'];
   private readonly checkPrivilegesFunc: CheckSavedObjectsPrivileges;
+  private readonly actionMap: Map<
+    SecurityAction,
+    { authzAction: string | undefined; auditAction: AuditAction | undefined }
+  >;
 
   constructor({ actions, auditLogger, errors, checkPrivileges }: Params) {
     this.actions = actions;
     this.auditLogger = auditLogger;
     this.errors = errors;
     this.checkPrivilegesFunc = checkPrivileges;
+
+    // This comment block is a quick reference for the action map, which maps authorization actions
+    // and audit actions to a "security action" as used by the authorization methods.
+    // Security Action                    ES AUTH ACTION          AUDIT ACTION
+    // -----------------------------------------------------------------------------------------
+    // Check Conflicts                    'bulk_create'           N/A
+    // Close PIT                          N/A                     AuditAction.CLOSE_POINT_IN_TIME
+    // Collect References                 'bulk_get'              AuditAction.COLLECT_MULTINAMESPACE_REFERENCES
+    // Collect Refs For Updating Spaces   'share_to_space'        AuditAction.COLLECT_MULTINAMESPACE_REFERENCES
+    // Create                             'create'                AuditAction.CREATE
+    // Bulk Create                        'bulk_create'           AuditAction.CREATE
+    // Delete                             'delete'                AuditAction.DELETE
+    // Bulk Delete                        'bulk_delete'           AuditAction.DELETE
+    // Find                               'find'                  AuditAction.FIND
+    // Get                                'get'                   AuditAction.GET
+    // Bulk Get                           'bulk_get'              AuditAction.GET
+    // Internal Bulk Resolve              'bulk_get'              AuditAction.RESOLVE
+    // Open PIT                           'open_point_in_time'    AuditAction.OPEN_POINT_IN_TIME
+    // Remove References                  'delete'                AuditAction.REMOVE_REFERENCES
+    // Update                             'update'                AuditAction.UPDATE
+    // Bulk Update                        'bulk_update'           AuditAction.UPDATE
+    // Update Objects Spaces              'share_to_space'        AuditAction.UPDATE_OBJECTS_SPACES
+    this.actionMap = new Map([
```

Comment:
I created this map because I was constantly referencing which audit action was used for an authorization action, and for which repo method. It is a bit verbose, but ended up being easier to manage when implementing more generic authorize and enforce methods.